### ISSUE TITLE
Earthly exits with "Error: parser failure: assignment to entry in nil map" when using EXPOSE keyword

### ISSUE THREAD

**Issue by cirego** [earthly/earthly#68]
I have an earthly build file that works until I add the folllowing line:

```
target:
    ...
    EXPOSE 8080
    ...
```
Adding this line causes Earthly to exit with the following message:

```
buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)
Error: parser failure: assignment to entry in nil map
```

**Comment by vladaionescu** [comment]
Hi @cirego - this was recently fixed in `v0.1.2`. Let me know if upgrading does not fix this.

**Comment by cirego** [comment]
Thanks for the quick reply. I will try upgrading!
